src/accordion/accordion-item.component.ts

```diff
@@ -36,7 +36,9 @@ export class AccordionItem {
 
 	@HostBinding("class.bx--accordion__item") itemClass = true;
 	@HostBinding("class.bx--accordion__item--active") @Input() expanded = false;
-	@HostBinding("style.display") @HostBinding("attr.role") itemType = "list-item";
+	@HostBinding("style.display") itemType = "list-item";
+	@HostBinding("attr.role") roleType = "heading";
```

Comment:
`itemType` makes sense as a variable name there, `roleType` should be `role`

Reply:
I'll make that change shortly